lib/ObjWriter/CMakeLists.txt

```diff
@@ -24,6 +24,9 @@ endif()
 add_library(objwriter
    SHARED
    objwriter.cpp
+   objwriter.h
```

Comment:
You should add a comment, this is the second time people have wondered why you are doing this.